### DISCUSSION TITLE
possibility for overflow model to generate character level alignments to the training dataset

### DISCUSSION THREAD

**Discussion by albluc24** [coqui-ai/TTS#2252] [Ideas]
Hello, I am very unfamiliar with the way the overflow model works, but I have seen that in it's dict of outputs it gives an alignments category. Now, I need a phoneme level aligner for my project and as far as I have seen it would be marvelous at the task because it learns some extremely good alignments, especially with phonemes. Is there a way to condition the alignments on the dataset's spectrogram so that it is possible to use a trained model as a forced aligner? If not, I have heard that for giving alignments to glow tts and such models, there is an aligner network that is jointly trained with those models. Is there a way to get dataset alignments out of that?
Thanks, have a good day!

**Comment by shivammehta25** [comment]
Hi! 
Since it is a left-to-right no skip, I would say it is better to have phoneme level alignments over character level alignments (which seems to be your case. So once the model is trained you can run a forward pass of the model and you can get alignments in `output['alignment']`.

Hope this helps!
